store/sqlstore/upgrade.go

```diff
@@ -497,6 +497,7 @@ func UpgradeDatabaseToVersion54(sqlStore SqlStore) {
 	if shouldPerformUpgrade(sqlStore, VERSION_5_3_0, VERSION_5_4_0) {
 		sqlStore.AlterColumnTypeIfExists("OutgoingWebhooks", "Description", "varchar(500)", "varchar(500)")
 		sqlStore.AlterColumnTypeIfExists("IncomingWebhooks", "Description", "varchar(500)", "varchar(500)")
+		sqlStore.CreateColumnIfNotExists("PluginKeyValueStore", "ExpireAt", "bigint(20)", "bigint", "0")
```

Comment:
We'll want to move this down into the `UpgradeDatabaseToVersion55` since it's too late to make it for 5.4

Reply:
Done, thanks.